libs/ui-toolkit/arrow-variation/src/lib/arrow-variation.component.ts

```diff
@@ -66,7 +70,12 @@ export class ArrowVariationComponent {
    *
    * @param n number to check
    */
+
   public updateArrow(n: number | '') {
+
+    if (this.pdfPrint) {
+       this.arrowLink = n > 0 ? '/assets/img/positive.png' : '/assets/img/negative.png';
```

Comment:
The value of `arrowLink` should always be updated in this function because `pdfPrint` and `value` are independent from each other. If you change `value` while `pdfPrint` is false, and then change `pdfPrint` to `true` afterwards, you'll use the arrow that corresponded to the previous value, not to the current one.